README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# duke.Duke project template
```

Comment:
Not sure if you want to include the package name in the `README.md` title folder. The change is possibly due to Intellij's refactoring